libs/designsystem/button/src/button.component.scss

```diff
@@ -254,11 +254,6 @@ $button-width: (
   --kirby-button-color: #{utils.get-color('success-contrast')};
 }
 
-:host-context(kirby-dropdown) {
-  // place arrow-icon in dropdown correctly
-  justify-content: space-between;
-}
-
```

Comment:
Removing this doesn't break any existing rendering?

Reply:
They do massively break the dropdown! I am going to revert them later and replace them with an appropriate dropdown rewrite.